tendrl/commons/logger.py

```diff
@@ -23,6 +23,9 @@ def __init__(self, message):
             self._logger(self.push_operation())
         else:
             self.push_event()
+            if self.message.priority == "notice":
+                # push notification
+                self.push_notification()
```

Comment:
Please return here, no need to further duplicate the notification in /messages/events